packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -340,70 +340,98 @@ describe('logs', () => {
     })
   })
 
-  describe('error logs limitation', () => {
+  describe(`logs limitation`, () => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏template string not needed
```suggestion
  describe('logs limitation', () => {
```